http_proxy.go

```diff
@@ -204,7 +204,7 @@ func (hp *HTTPProxy) upstreamProxyURL() *url.URL {
 }
 
 func (hp *HTTPProxy) pacProxy(r *http.Request) (*url.URL, error) {
-	s, err := hp.pac.FindProxyForURL(r.URL, r.Host)
+	s, err := hp.pac.FindProxyForURL(r.URL, "")
```

Comment:
nit: maybe a short comment on why this is empty

Reply:
This is documented in ProxyResolver. I updated the docs and copied them to the interface in forwarder.